docs/app/components/api-property/api-property.component.html

```diff
@@ -0,0 +1,12 @@
+<td class="api-property col-md-3">
+    <span class="highlight">{{ name }}</span>
+</td>
+<td class="api-property col-md-9">
```

Comment:
Both of these tds have the same class name. It would be helpful if they were different, for purposes of targeting with CSS, e.g. `api-property-name` and `api-property-details`.